src/main/java/com/answerdigital/answerking/controller/CategoryController.java

```diff
@@ -67,19 +67,19 @@ public ResponseEntity<Collection<Category>> getAllCategories() {
         return new ResponseEntity<>(categories, categories.isEmpty() ? HttpStatus.NO_CONTENT : HttpStatus.OK);
     }
 
-    @Operation(summary = "Get a single category")
+    @Operation(summary = "Get a single category.")
     @ApiResponses(value = {
         @ApiResponse(responseCode = "200", description = "When the category with the provided id has been found.",
                 content = { @Content(mediaType = "application/json", schema = @Schema(implementation = Category.class)) }),
-        @ApiResponse(responseCode = "404", description = "When the category with the given id does not exist",
+        @ApiResponse(responseCode = "404", description = "When the category with the given id does not exist.",
```

Comment:
This isn't aligned with the spec, but I'd argue this is a fault with the spec.

Reply:
for this docs we will have to address it in a different PR.